packages/client/src/flow/proposeNFTTransfer.tx.js

```diff
@@ -3,7 +3,7 @@ export const PROPOSE_NFT_TRANSFER = `
 	import DAOTreasuryV4 from 0xDAOTreasuryV4
 	import NonFungibleToken from 0xNonFungibleToken
 	import MyMultiSigV4 from 0xMyMultiSigV4
-	import ExampleNFT from 0xExampleNFT
+	import ZeedzINO from 0xZeedzINO
```

Comment:
looks like this one also needs to be generic. https://github.com/DapperCollectives/VESSEL/blob/main/packages/client/src/hooks/useNFTs.js#L34.
Probably it's beyond the scope of your original ticket, is it possible to accommodate in this PR?

Reply:
Updated this one too, thank you! :)